server_src/routes/printers.js

```diff
@@ -98,14 +102,12 @@ router.post("/feedChange", ensureAuthenticated, async (req, res) => {
 });
 router.post("/updateSettings", ensureAuthenticated, async (req, res) => {
   // Check required fields
-
   const settings = req.body;
   logger.info("Update printers request: ", settings);
   const updateSettings = await Runner.updateSettings(settings);
-
   res.send({ status: updateSettings.status, printer: updateSettings.printer });
 });
-router.get("/killPowerSettings/:id", ensureAuthenticated, async (req, res) => {
+router.post("/killPowerSettings/:id", ensureAuthenticated, async (req, res) => {
```

Comment:
Updated to a post command as backend is changed

Reply:
I dont understand this comment as this is the backend we're looking at?